_ont/ont-huawei-ma5671a-fs-mod.md

```diff
@@ -0,0 +1,188 @@
+---
+title: FS Modded Firmware for Huawei MA5671A and FS.com GPON-ONU-34-20BI
+has_children: false
+parent: Huawei MA5671A
+layout: default
+---
+
+# Flashing the firmware
+
+Flash the firmware image to image0 and image1 via [SSH](/ont-huawei-ma5671a/#flashing-a-new-rootfs-via-ssh) or via [TTL serial](/ont-huawei-ma5671a-ymodem) as usual.
+
+After the flash it is absolutely necessary to connect to the SFP via telnet using the IP 192.168.1.10:
+
+```sh
+telnet 192.168.1.10
+```
+
+Once connected via telnet to the SFP, execute the following commands, it will be necessary to wait a few minutes until the end of the automatic procedure:
+
+```sh
+firstboot
+reboot
+```
+
+Remember that with each flash of this firmware it is always necessary to perform the procedure described above, otherwise the SFP will not work!
+
+# Login info
+|                  |                                                |
+| ---------------- | ---------------------------------------------- |
+| Username         | root                                           |
+| Password         | root                                           |
+| IP address	     | 192.168.1.10                                   |
+| Telnet    	     | Always available                               |
+| TTL Serial 	     | Always available                               |
+| SSH       	     | Only after the first configuration (firstboot) |
+| Web       	     | NO                                             |
+
+# GPON ONU status
```

Comment:
Is there a specific reason why 'General Settings and Useful Commands' was not used as the title as on most other pages?

Reply:
No, it's just more comfortable this way

Reply:
@nanomad opinions?